docs/histogram.md

```diff
@@ -6,11 +6,11 @@ Analyze the value distribution of a single continuous variable by binning it and
 
 ## Parameters
 
-|       Name        |    Type     |                                  Description                                   | Is Optional |
-| ----------------- | ----------- | ------------------------------------------------------------------------------ | ----------- |
-| column            | column      | numeric column to use to generate the histogram                                |             |
-| filter_statements | string_list | List of SQL where statements to filter the table by, i.e. 'COLUMN IS NOT NULL' | True        |
-| num_buckets       | value       | max number of buckets to create; defaults to 200                               | True        |
+|       Name        |    Type     |                   Description                    | Is Optional |
+| ----------------- | ----------- | ------------------------------------------------ | ----------- |
+| column            | column      | numeric column to use to generate the histogram  |             |
+| filter_statements | filter_list | list of dictionaries representing filters        | True        |
```

Comment:
can we name this input 'filters' everywhere instead of filter_statements?

Reply:
also sorry for the nitpick but let's change the description everywhere too since it will show up on the hover tooltip in UI:
'Filter logic on one or more columns. Can choose between a simple comparison filter or advanced filter using free text.'